ihatemoney/utils.py

```diff
@@ -14,6 +14,32 @@
 
 
 import csv
+import requests
+
+
+class CurrencyConverter(object):
+    # Convert all to USD
+    api_url = "https://api.exchangerate-api.com/v4/latest/USD"
+    response = []
+
+    def __init__(self):
+        self.response = requests.get(self.api_url).json()
+
+    def get_currencies(self):
+        currencies = []
+        for rate in self.response["rates"]:
+            currencies.append((rate, rate))
+        return currencies
+
+    def exchange_currency(self, amount, source_currency, dest_currency):
```

Comment:
Thanks for taking this back to life :-) It would be great to cache the rates on disk, rather than doing the request all the time. It would also allow to save us in case the service is down. Let's say we'll cache the rates once a day per currency, if that's okay for you!

Reply:
Hey, sounds great, I'll find a python package for it!

Reply:
@almet please have a look now. I added a on disk cache.

Reply:
That looks great :-) I'm just a bit reluctant to have all this in utils.py now. What about creating a `currency_convertor.py` file and putting all the logic in it, as well as instantiating the convertor object in the setup of the application, passing it the cache as an argument, for instance ?

Reply:
@almet happy to change that. Give me some more time.

Reply:
@almet please have another look.